tests/eips/eip4844/test_blob_txs.py

```diff
@@ -447,13 +449,20 @@ def test_invalid_tx_max_fee_per_data_gas(
     pre: Dict,
     env: Environment,
     blocks: List[Block],
+    parent_blobs: int,
+    block_intermediate: Block,
 ):
     """
     Reject blocks with invalid blob txs due to:
 
     - tx max_fee_per_data_gas is barely not enough
     - tx max_fee_per_data_gas is zero
     """
+    if parent_blobs is not None:
```

Comment:
We could reduce it to:
```suggestion
    if parent_blobs:
```
to check both `not None` and `> 0`.